src/bot.ts

```diff
@@ -0,0 +1,36 @@
+import { Message } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+import { CONFIG } from './global';
+
+class Bot extends CommandoClient {
+  constructor() {
+    super({
+      commandPrefix: CONFIG.prefix,
+    });
+  }
+
+  public async start() {
+    this.registry
+      .registerDefaultTypes({
+        textChannel: true,
+      })
+      .registerGroup('staff')
+      .registerCommandsIn(`${__dirname}/commands`);
+    this.on('message', this.onMessage.bind(this));
+    await this.login(CONFIG.token);
+  }
+
+  public async onMessage(msg: Message) {
```

Comment:
For JavaScript comments we use "JSDoc" which looks something like this
```js
/**
 * <description>
 * @param {ParamType} paramName
 * @returns {ReturnType}
 */
```

So for this method it would look like this:
```js
/**
 * Message event handler
 * @param {Message} msg
 * @returns {Promise<void>
 */
```

You can learn more about jsdoc [here](https://jsdoc.app/)